src/components/Charts/chart-defaults.js

```diff
@@ -0,0 +1,112 @@
+import {defaults} from 'react-chartjs-2';
+import deepmerge from 'deepmerge';
+
+defaults.global = deepmerge(defaults.global, {
+  defaultFontFamily: 'Archia',
+  elements: {
+    line: {
+      fill: false,
+    },
+    point: {
+      pointStyle: 'rectRounded',
+    },
+  },
+  tooltips: {
+    intersect: false,
+    mode: 'nearest',
+    position: 'average',
+    displayColors: false,
+    borderWidth: 0,
+    titleSpacing: 4,
+    titleMarginBottom: 8,
+    bodySpacing: 8,
+    cornerRadius: 3,
+    xPadding: 10,
+    yPadding: 8,
+    caretSize: 0,
```

Comment:
I feel tooltip is too big now and some part hides behind bottom of the chart. Need to revisit this.

Reply:
While the current tooltip is concise, I did find issues when trying to actually read the contents with small text on a semi-transparent background.

![image](https://user-images.githubusercontent.com/8946207/79997364-1f870980-84d7-11ea-9259-4f7b97cb8335.png)
![image](https://user-images.githubusercontent.com/8946207/79997373-21e96380-84d7-11ea-9b52-3230d2aba599.png)



I've updated this to use tighter spacing now (still a tad little roomier than exists today)